src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1296,29 +1293,36 @@ By default, we include the following package managers in each runtime.
 {% /table %}
 
 # Using Appwrite in a function {% #using-appwrite %} 
-Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies. 
-Authenticating with Appwrite is done via an API key or a JWT token. 
-API keys must be generated and exported as an [environment variable](/docs/advanced/self-hosting/environment-variables).
+Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies.
+Integrate with Appwrite using its authentication, which is done via an API key, a JWT token, or a dynamic API key. 
+Using dynamic API keys, you can easily integrate with Appwrite.
```

Comment:
Don't mention API keys. We want to encourage using dynamic API keys so we should only talk about that.